node-tests/fixtures/helper-test/rfc232-unit.js

```diff
@@ -0,0 +1,12 @@
+import { fooBarBaz } from 'my-app/helpers/foo/bar-baz';
```

Comment:
@rwjblue should we look this up on the container instead?

Reply:
https://github.com/rwjblue/ember-qunit-codemod/blob/master/__testfixtures__/ember-qunit-codemod/native-qunit-to-nested.output.js would suggest no
but I guess in the above scenarios we aren't really testing the helper itself, just the underlying function.

I have found it kind of hard to test class based helpers -- for example:
```js
this.render(hbs`{{my-helper data}}`);
```
may return objects that don't serialize well.


Would be great if we could do something like
```js
this.render(hbs`{{let actualResult (my-helper data)}}` );
assert.equal(this.actualResult, expectedResult);
```

Reply:
If you're returning something other than a string or number you should probably not use a rendering test and use a "unit" test instead. In any case both tests should probably use the container. Rendering tests do that implicitly and unit tests explicitly.

see also https://github.com/rwjblue/ember-qunit-codemod/issues/17

Reply:
@Turbo87 should I change the unit test to use the container then using `factoryFor` then?

Reply:
Not right now, lets focus on 1:1 for now then we can follow up with a PR that adds usage of `owner.factoryFor('helper:foo').create()`.

Reply:
I've discussed this at length with @krisselden and I believe he is opposed to the public blueprints instantiating helpers outside of the confines of the template layer.  I personally disagree, but I don't want to make a change and "sneak it in" without a more thorough discussion.  Does that make sense?